client/src/app/LandingPage.css

```diff
@@ -0,0 +1,49 @@
+.sec1-container{
+    display: flex;
+    width: 100%;
```

Comment:
if you're doing flex, the width will already expand to fill in the width of it's parent's container